package.json

```diff
@@ -6,26 +6,76 @@
   "description": "Tests for the New Relic JavaScript agent",
   "license": "Apache-2.0",
   "homepage": "https://docs.newrelic.com/docs/browser/browser-monitoring",
-  "main": "",
+  "main": "./dist/cjs/index.js",
   "types": "./types.ts",
-  "module": "./dist/mjs/index.mjs",
+  "module": "./dist/esm/index.js",
```

Comment:
`module` field proposal rejected (superceded by another proposal field `browser`) -- 
Can be removed with newer `exports` taking precedence over `main` (Node <=v10) / `module` / `browser` ~esbuild.github.io/api/#main-fields

Reply:
Does this still need resolution after the discussion earlier?

Reply:
I think module was just included for bundlers that utilize it, but since we are sticking to requiring node versions that support export, I'm not opposed to removing it. I will just do that for now